packages/translations/src/i18next/i18next.js

```diff
@@ -41,10 +60,10 @@ const isLanguageAvailable = lang => {
 
     if (is_ach) return isStaging() || isLocal();
 
-    return Object.keys(ALL_LANGUAGES).includes(selected_language);
+    return Object.keys(getAllLanguages()).includes(selected_language);
 };
 
-export const getAllLanguages = () => ALL_LANGUAGES;
+export const getAllLanguages = () => (isProduction() ? allowed_languages : ALL_LANGUAGES);
```

Comment:
I think it might be better to return all languages here and instead use `getAllowedLanguages` on line 63 since that function already handles this condition on line 38

Reply:
```suggestion
export const getAllLanguages = () => ALL_LANGUAGES;
```